docs/content/configuration/broker.md

```diff
@@ -103,3 +103,4 @@ You can optionally only configure caching to be enabled on the broker by setting
 |`druid.cache.hosts`|Command separated list of Memcached hosts `<host:port>`.|none|
 |`druid.cache.maxObjectSize`|Maximum object size in bytes for a Memcached object.|52428800 (50 MB)|
 |`druid.cache.memcachedPrefix`|Key prefix for all keys in Memcached.|druid|
+|`druid.cache.failureMode`|Failure mode when there is a problem communicating with memcached(`Cancel` or `Retry`)|`Cancel`|
```

Comment:
Yes, it is exactly the value of the num. I don't see a reason to have it not be the value of the enum.